.github/stale.yml

```diff
@@ -0,0 +1,25 @@
+# MRTK stale configuration
+# In order to ensure that the issues on Github remain somewhat fresh
+# and update to date (i.e. aren't tracking releases from over a year)
```

Comment:
Seems like this `i.e.` continues on the next line, even though it ends here with a `)`

Reply:
Yep, also botched a few words